INTEGRATION.md

```diff
@@ -0,0 +1,166 @@
+# Gateway Integration
+
+![wip](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+**Author(s)**:
+- [Mark Gaiser](https://github.com/markg85/)
+
+**Maintainer(s)**:
+- [Mark Gaiser](https://github.com/markg85/)
+
+* * *
+
+## Summary
+
+Defines the decision model an IPFS enabled application should use to find a IPFS gateway. Simultaneously defines how an IPFS implementation should expose it's gateway.
+
+For clarity upon reading this document:
+**IPFS implementation**, this is an application implementing (a set of) the IPFS specifications. It would expose a gateway for other applications to use. Examples here are KUBO and Iroh.
+
+**IPFS integration**, this is an application integrating the IPFS protocol support. Meaning that an application "integrating IPFS" can fetch IPFS resources. These applications often are using gateways to implement IPFS support. Examples here are ffmpeg and curl.
+
+## Motivation
+
+Applications wanting to use IPFS resources are, without this spec, left to invent their own ways of finding a gateway. This spec defines how an application wanting to implement IPFS support can find (at least one) gateway to use.
+
+Simultaneously the spec also defines how IPFS implementations should expose their gateway.
+
+## Detailed design
+
+This integration spec defines the recommended way for a third party applications to integrate IPFS support in their application and thereby gaining easy access to resources stored on the IPFS platform.
```

Comment:
s/stored/available/

ipfs is address-ability but not necessarily storage. short-caching http gateways are a great example of that.